tests/sentry/lang/native/test_applecrashreport.py

```diff
@@ -469,7 +536,7 @@ def test__convert_debug_meta_to_binary_image_row():
             "image_addr": "0xd69a000",
             "image_size": 495616,
             "image_vmaddr": "0x0",
-            "name": "/Users/haza/Library/Developer/CoreSimulator/Devices/DDB32F4C-97CF-4E2B-BD10-EB940553F223/data/Containers/Bundle/Application/8F8140DF-B25B-4088-B5FB-57F474A49CD6/SwiftExample.app/Frameworks/SentrySwift.framework/SentrySwift",
+            "code_file": "/Users/haza/Library/Developer/CoreSimulator/Devices/DDB32F4C-97CF-4E2B-BD10-EB940553F223/data/Containers/Bundle/Application/8F8140DF-B25B-4088-B5FB-57F474A49CD6/SwiftExample.app/Frameworks/SentrySwift.framework/SentrySwift",
```

Comment:
Sorry for the confusion, that was not the only schema change.

Could you, for each place where you rename `name` to `code_file` in tests:

* change type to `native`
* remove `cpu_type` and `cpu_subtype` if present